src/index.js

```diff
@@ -80,3 +112,42 @@ listContainer.addEventListener('click', (e) => {
     completed(e.target, todoLists);
   }
 });
+
+// add the task
+
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const todoInput = document.getElementById('myInput').value;
+  if (todoInput === '') {
+    alert('Please, enter the To Do for the day');
+  } else {
+    addTodoToList(todoInput, false, todoLists.length, todoLists);
+    addTodo();
+    saveToStorage(todoLists);
+    form.reset();
+  }
+});
+
+// delete the task
+
+listContainer.addEventListener('click', (e) => {
+  const task = document.querySelector('.items').textContent;
+  if (e.target.classList.contains('list-container')) {
+    deleteTodoList(e.target); // ui;
+  }
+  if (e.target.classList.contains('delete')) {
+    todoLists = todoLists.filter((item) => item.description !== task);
+    reindex(todoLists);
+    // saveToStorage(todoLists);
```

Comment:
# 💾 💾 💽 [OPTIONAL]   💽 💾 💾
**✔️ _Keep professionalism in your Repo's_** 
- ⏏︎ Remove commented code from your project.